frontend/server/src/DAO/CoderOfTheMonth.php

```diff
@@ -86,14 +88,14 @@ public static function calculateCoderOfTheMonth(
             ) AS cm on i.user_id = cm.user_id
           WHERE
             (cm.user_id IS NULL OR DATE_ADD(cm.latest_time, INTERVAL 1 YEAR) < ?) AND
-            i.user_id IS NOT NULL
+            i.user_id IS NOT NULL " . $gender . '
```

Comment:
```suggestion
            i.user_id IS NOT NULL {$genderClause}
```

así no necesitas cerrar y abrir la cadena!